scripts/actions/repl.js

```diff
@@ -0,0 +1,84 @@
+const COMMENT_PREFIX =
+  "Build successful! You can test your changes in the REPL here: ";
+
+export default async function (github, context) {
+  const { owner, repo } = context.repo;
+  const pull_number = context.issue.number;
+
+  let commentId;
+
+  const comments = await github.rest.issues.listComments({
+    owner: owner,
+    repo: repo,
+    issue_number: pull_number,
+  });
+  for (const comment of comments.data) {
+    if (comment.body.includes(COMMENT_PREFIX)) {
+      commentId = comment.id;
+      break;
+    }
+  }
+
+  // Make sure Circleci is triggered.
+  await sleep(60000);
```

Comment:
I think instead we can have a GH workflow triggered by the CircleCI workflow, but this is ok for now